simplify/src/main/java/org/cf/simplify/Launcher.java

```diff
@@ -179,33 +179,44 @@ private void executeClass(VirtualMachine vm, String className, RuntimeStats stat
                 continue;
             }
 
-            boolean executeAgain;
-            do {
-                System.out.println("(" + stats.getCurrentMethodIndex() + " / " + stats.getTotalMethods() + ") Executing top level method: " + method);
-                ExecutionGraph graph = null;
-                try {
-                    graph = vm.execute(method);
-                } catch (VirtualMachineException e) {
-                    System.err.println("Aborting execution; exception: " + e);
-                }
-
-                if (null == graph) {
-                    System.out.println("Skipping optimization of " + method + "; null execution graph");
-                    stats.incrementFailedMethodCount();
-                    break;
-                }
+            try {
```

Comment:
👍 Hallo.

I agree it's better to fix the cause, but it's hard. This PR is good because it fixes an issue a few people have mentioned but I've put off fixing. Thanks for getting it started.

It should be easy to re-create the VM on crash. I think this would make a better default behavior. If you're up for it (and you agree), feel free to add it.